abcd/server/app/views/api.py

```diff
@@ -1,21 +1,20 @@
 from flask import Blueprint, Response, make_response, jsonify, request
 
-bp = Blueprint('api', __name__)
+bp = Blueprint("api", __name__)
 
 
-@bp.route('/')
+@bp.route("/")
 def index():
-    return Response('ok', status=200)
+    return Response("ok", status=200)
```

Comment:
[Python HTTP](https://docs.python.org/3/library/http.html) has nice aliases for these